README.md

```diff
@@ -113,7 +127,8 @@ end
 Your answer:
 ```text
 
-Replace this with your answer
+describe is used for classes and methods while context is used for conditionals
```

Comment:
:white_check_mark: , although not necessarily just for conditionals -- `context` is more accurately for **situations**, as in:

```rb
describe "#say_hello"
  context "when frog is in throat"
```